cloud_build/deploy.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+  - name: gcr.io/cloud-builders/git
+    args: ['submodule', 'update', '--init', '--recursive']
+  - name: gcr.io/cloud-builders/docker
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - |-
+        set -e
+        echo "Building the website using a makefile..."
+        make build
+  - name: gcr.io/flutter-dev-230821/firebase-ghcli
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - |-
+        firebase deploy --project=flutter-website-staging-349021 --only=hosting
+options:
```

Comment:
The other ones you two added had to specify what secrets were available: 

https://github.com/dart-lang/site-www/blob/2b4e25631c9d9cd194febe98fc04f575b403cb0d/cloud_build/stage.yaml#L26-L29

Is that not necessary in this case?

Reply:
Nope, the GH_PAT_TOKEN secret is only required for the staging configuration because it needs to add a comment back to the GitHub PR with a staging link. This is just a one way deploy to Firebase Hosting so no GitHub access needed.

Reply:
That makes a lot of sense...

Well this senseless developer say thanks for the explanation :)

Reply:
It made more sense to me _after_ I wrote out that explanation so thanks for asking! hehe